integration_tests/tests/test_exposure_schema_validity.py

```diff
@@ -0,0 +1,161 @@
+from datetime import date, timedelta
+from typing import Any, Dict, List
+
+from data_generator import DATE_FORMAT, generate_dates
+from dbt_project import DbtProject
+
+DBT_TEST_NAME = "elementary.exposure_schema_validity"
+
```

Comment:
Can we maybe add one test that relies on "real" exposures (rather than dependency injection)?
My main concern is that the format of exposures in the graph will change between dbt versions and the test will miss it.

I think you can just commit models / exposures that you need to the test dbt project.